### DISCUSSION TITLE
Using the Slack events API

### DISCUSSION THREAD

**Discussion by sornas** [lithekod/kodapa#34] [Ideas]
Do we want this? We'd need to handle webhooks in some way.

**Comment by FredTheDino** [comment]
1. How much code would it take?
2. Do we get any form of benefit from it?
3. Do we want to?
With me I mean you.

**Reply by sornas** [reply]
1. Can't say really. We'd need [hyper](https://github.com/hyperium/hyper/blob/master/examples/single_threaded.rs) to handle requests which doesn't look too difficult, and we'd probably have about the same code size as now.
2. No Slack sockets.
3. Yeah.
